content/node-runners/postinstall/index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Post install configuration
+weight: 10
+---
+
+### Startup configuration
+The installed DEB package will be starting a node service automatically.
+
+You can check details using the following command:
+
+```shell
+sudo systemctl status mysterium-node.service
+```
+
+You can change the configuration of the service via `/etc/default/mysterium-node` file:
+
+```shell
+root@raspberrypi:~# cat /etc/default/mysterium-node
+# Define additional args for `myst` service (see `myst --help` for full list)
+CONF_DIR="--config-dir=/etc/mysterium-node"
+RUN_DIR="--runtime-dir=/var/run/mysterium-node"
+DATA_DIR="--data-dir=/var/lib/mysterium-node"
+DAEMON_OPTS="--tequilapi.address=0.0.0.0"
+SERVICE_OPTS="openvpn"
+```
+
+List of available options can be found using a help commands: `myst --help` or `myst service --help`.
+
+For example, if you want to start only OpenVPN service on 1190 port and serve Mysterium verified consumer only, change the `SERVICE_OPTS=` line to the following:
+
+```shell
+SERVICE_OPTS="--openvpn.port=1190 --access-policy.list=mysterium openvpn"
+```
+
+To apply the changed service configuration you will need to restart service.
+
+```shell
+sudo systemctl restart mysterium-node
+```
+
+### TequilAPI and CLI
+Runtime configuration of the running Mysterium node can be changed using a [TequilAPI](https://tequilapi.mysterium.network/) or CLI commands.
+
+**TequilAPI** is a REST API that allows you to manipulate a Mysterium node multiple ways.
+
+```
+curl http://localhost:4050/healthcheck
```

Comment:
I think you should use [node's internal IP] rather than 'localhost'. People are used to centralised APIs, we should mention, that each node is exposing own API, this API is accessible either on same machine (via localhost) or from internal network (e.g. when running on docker).